src/main/kotlin/no/skatteetaten/aurora/boober/feature/ToxiproxySidecarSpec.kt

```diff
@@ -50,10 +52,29 @@ fun AuroraDeploymentSpec.extractToxiproxyEndpoints(): List<Pair<String, String>>
 // To be used when there is no proxy name specified by the user
 fun generateProxyNameFromVarName(varName: String) = "endpoint_$varName"
 
-fun List<ToxiProxyConfig>.findPortByProxyName(proxyName: String) = this
-    .find { it.name == proxyName }
-    ?.listen
-    ?.substringAfter(':')
+// Regex for matching a proxy name in a server and port field name
+val proxyNameInServerAndPortFieldNameRegex =
+    Regex("(?<=^toxiproxy\\/serverAndPortFromConfig\\/)([^\\/]+(?=\\/serverVariable\$|\\/portVariable\$|\$))")
```

Comment:
Er dette dekket av en test?

Reply:
Den dekkes av testen `Should add toxiproxy to dc and map servers and ports from container`, men jeg kan godt lage en spesifikk test for akkurat dette for å dekke litt flere brukstilfeller.